### PR TITLE
Dom/bin ordinal docs

### PR DESCRIPTION
**Pull request by domoritz**

### CODE REVIEW

File: _data/link.yml [hunk 1]
```diff
@@ -15,8 +15,9 @@ Transform:
 
 AggregateOp:
   link: "aggregate.html#ops"
-Bin:
-  link: bin.html
+BinParams:
+  name: Bin
```
Comment (kanitw) [inline _data/link.yml]: `Bin` is `boolean | BinParams` so this technically undoes what I intended to change.  Please remove this line.

File: site/docs/bin.md [hunk 2]
```diff
@@ -91,3 +91,9 @@ If `bin` is `true`, default binning parameters are used. To customize binning pa
 Setting the `maxbins` parameter changes the number of output bins.
 
 <div class="vl-example" data-name="histogram_bin_change"></div>
+
+### Ordinal Bin
```
Comment (kanitw) [inline site/docs/bin.md]: Should this be `##`  (h2)?

File: _data/link.yml (hunk 1 above)
Comment (domoritz) [inline _data/link.yml]: Done

File: site/docs/bin.md (hunk 2 above)
Comment (domoritz) [inline site/docs/bin.md]: Done

**Pull request merged**